### DISCUSSION TITLE
How to get API key?

### DISCUSSION THREAD

**Discussion by TheZoker** [kimai/kimai#4833] [Questions & Answers]
Hi there,

I created a API key and it is showing in the dashboard. But how do I get the key itself? 
When I created it, the popup is closing and the new name of key is shown in the API key list. But the information about the key itself is missing.

**Comment by kevinpapst** [comment]
Thanks for sharing 👍 

https://github.com/kimai/kimai/pull/4828 - will release it soon as bugfix release

**Comment by kevinpapst** [comment]
Closing, fixed in 1.16.1
